### DISCUSSION TITLE
npm run build SvelteKit don't push also tailwindcss in the ./package/ of a component library (npm package)

### DISCUSSION THREAD

**Discussion by Laaouatni** [tailwindlabs/tailwindcss#10410] [Help] [answered]
**What version of Tailwind CSS are you using?**

For example: latest

**What build tool (or framework if it abstracts the build tool) are you using?**

SvelteKit

**What version of Node.js are you using?**

I don't use node

**What browser are you using?**
**What operating system are you using?**

```Shell
System:
    OS: Windows 10 10.0.22621
    CPU: (8) x64 11th Gen Intel(R) Core(TM) i5-1135G7 @ 2.40GHz
    Memory: 1.04 GB / 7.65 GB
  Binaries:
    Node: 16.14.0 - C:\Program Files\nodejs\node.EXE
    npm: 9.3.1 - C:\Program Files\nodejs\npm.CMD    
  Browsers:
    Edge: Spartan (44.22621.1105.0), Chromium (109.0.1518.55)
    Internet Explorer: 11.0.22621.1
  npmPackages:
    @sveltejs/adapter-auto: ^1.0.0 => 1.0.2
    @sveltejs/kit: ^1.0.0 => 1.2.6
    @sveltejs/package: ^1.0.0 => 1.0.2
    svelte: ^3.54.0 => 3.55.1
    vite: ^4.0.0 => 4.0.4
```

### Describe the bug

basically, I was creating a simple library with sveltekit and using tailwindcss. the problem is when I do "npm run build" it creates the files correctly but when i do npm install of the same file. the component don't have any CSS applied to it.

### Reproduction

my npm package: https://www.npmjs.com/package/mathview-svelte
my GitHub repo:   https://github.com/Laaouatni/MathViewSvelte

example of the error: https://svelte.dev/repl/39421699a03f4717a74e6665f7055eff?version=3.55.1

what should output:
![image](https://user-images.githubusercontent.com/87947051/214435105-17b3e031-084c-4d18-a68c-bb2b70378ba5.png)

what is wrong:

![image](https://user-images.githubusercontent.com/87947051/214435186-ac9a827f-8f8e-455e-a5a0-b58399bdd4ac.png)

another info is that tailwind works fine in dev mode "npm run dev", but when build don't work fine.

how make also the css being exported with the component library?

### System Info

```Shell
System:
    OS: Windows 10 10.0.22621
    CPU: (8) x64 11th Gen Intel(R) Core(TM) i5-1135G7 @ 2.40GHz
    Memory: 1.04 GB / 7.65 GB
  Binaries:
    Node: 16.14.0 - C:\Program Files\nodejs\node.EXE
    npm: 9.3.1 - C:\Program Files\nodejs\npm.CMD    
  Browsers:
    Edge: Spartan (44.22621.1105.0), Chromium (109.0.1518.55)
    Internet Explorer: 11.0.22621.1
  npmPackages:
    @sveltejs/adapter-auto: ^1.0.0 => 1.0.2
    @sveltejs/kit: ^1.0.0 => 1.2.6
    @sveltejs/package: ^1.0.0 => 1.0.2
    svelte: ^3.54.0 => 3.55.1
    vite: ^4.0.0 => 4.0.4
```

### Severity

annoyance

**Comment by RobinMalfait** [comment, marked as answer]
Hey! Thank you for your question!
Much appreciated! 🙏

You either have to make sure that the application that uses your package also uses tailwind and includes the files from your package in the `content` of your tailwind.config.js file.

Or, you can build the final css file in your package itself, but then you have to make sure that:
- You publish your css file to `npm` in your package
- You include the generated css file in your project as well

Hope this helps!

**Reply by Laaouatni** [reply]
I solved the problem by adding to the content array of `tailwind.config.js` this line of code: `"./node_modules/**/*.svelte"`
